src/hud/elements/ExampleElement.ts

```diff
@@ -3,5 +3,28 @@ import {HudElement} from "../HudElement";
 export class ExampleElement extends HudElement {
 	constructor() {
 		super("ExampleElement");
+
+		this.setCoordinates(100, 100);
+		document.addEventListener("keydown", (event) => {
```

Comment:
ui elements shouldn't listen to global events

Reply:
Yeah, this was just for the sake of the example. Probably shouldn't have committed it.